core/controllers/reader.py

```diff
@@ -1032,16 +1031,14 @@ def get(self):
             raise self.InvalidInputException(
                 'Question count has to be greater than 0')
 
-        questions, _, next_start_cursor = (
-            question_services.get_questions_and_skill_descriptions_by_skill_ids(
+        questions = (
```

Comment:
Can you remind me again why are we removing cursor completely? This is because pretests, review tests, and practice sessions will not be paginating right?

Reply:
Yes. There will be no pagination in question players, so cursors won't be used at all.